### DISCUSSION TITLE
Remote Broadcast

### DISCUSSION THREAD

**Discussion by amoscatelli** [smallrye/smallrye-reactive-messaging#2055] [Q&A] [answered]
Is it possible to achieve a publish/subscribe remote broadcast using this spec ?
I am **not** talking about local - inmemory - broadcast

I am trying different configurations but I am getting only 1 consumer to process a single message in my cloud deployment.

Thank you in advance

**Comment by cescoffier** [comment]
Yes, but it depends on the connector and often on the broker configuration.

**Comment by amoscatelli** [comment]
Can you make an example ?

I am trying to use RabbitMQ, but it seems I can't achieve what I want since I need a single fanout exchange but multiple (dynamic I suppose?) queues.

What connector would you suggest ?

**Comment by amoscatelli** [comment]
Of course I can configure the single fanout exchange, the issue is related to the dynamic queues since it seems queue.name is required and static ...

**Comment by amoscatelli** [comment, marked as answer]
I found the solution : 

mp.messaging.incoming.chat-incoming.queue.name=${HOSTNAME}
mp.messaging.incoming.chat-incoming.queue.auto-delete=true
mp.messaging.incoming.chat-incoming.exchange.name=exchangename
mp.messaging.incoming.chat-incoming.exchange.type=fanout
mp.messaging.outgoing.chat-outgoing.exchange.name=exchangename
